opensalt/opensalt
# Use server-provided value for "Case Framework URL"

Closes #568
Closes #471

Related #563

Updates client-side code to display the server-provided `uri` property value for "Case Framework URL".

**Test plan**

Manual testing:

1. Import the attached CASE file
2. Browse to the "Test Framework with Imported URIs" Framework
3. Expect that the Document's "Case Framework URL" is **http://localhost:3000/uri/https://www.testframeworks.org/doc/1**
4. Expect that the Item's "CASE Item URI" (under "More Info") is **http://localhost:3000/uri/https://www.testframeworks.org/item/1**

[case-format-with-uri.json.zip](https://github.com/opensalt/opensalt/files/2287979/case-format-with-uri.json.zip)

![screen shot 2018-08-14 at 5 08 13 pm](https://user-images.githubusercontent.com/7999269/44118631-c9f30772-9fe4-11e8-9ebb-ae59eb98e2be.png)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/opensalt/opensalt/578)
